<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QGroupBox" name="generalGroupBox">
     <property name="title">
      <string>General Settings</string>
     </property>
     <layout class="QVBoxLayout" name="generalLayout">
      <item>
       <layout class="QHBoxLayout" name="downloadFolderLayout">
        <item>
         <widget class="QLabel" name="downloadFolderLabel">
          <property name="text">
           <string>Default Download Folder:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="downloadFolderLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="autoStartCheckBox">
        <property name="text">
         <string>Auto-start new downloads</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="promptOverwriteCheckBox">
        <property name="text">
         <string>Prompt before overwriting existing files</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="namingPolicyLayout">
        <item>
         <widget class="QLabel" name="namingPolicyLabel">
          <property name="text">
           <string>File Naming Policy:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="namingPolicyComboBox">
          <item>
           <property name="text">
            <string>Use original name</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Append number if exists</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Append timestamp</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="maxDownloadsLayout">
        <item>
         <widget class="QLabel" name="maxDownloadsLabel">
          <property name="text">
           <string>Max Concurrent Downloads:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="maxDownloadsSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
